twitterscraper/tweet.py

```diff
@@ -17,27 +17,34 @@ def __init__(self, user, fullname, id, url, timestamp, text, replies, retweets,
         self.retweets = retweets
         self.likes = likes
         self.html = html
+        self.mentions = mentions
```

Comment:
You are never passing the contents of 'mentions' in the from_soup method to this class. (see line 9)